### PR TITLE
added azimuthal correlation analysis

### PR DESCRIPTION
**Pull request by jaimenorman**
Added histograms to fill for azimuthal distributions

### CODE REVIEW

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetCoreEmcal.h [hunk 1]
```diff
@@ -111,6 +113,11 @@ class AliAnalysisTaskJetCoreEmcal : public AliAnalysisTaskEmcalJet {
 	TH2F *fh2RPTC10; //!
 	TH2F *fh2RPTC20; //!
 
+	TH2F *fhTTPt; //!
```
Comment (raymondEhlers) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetCoreEmcal.h]: I'd suggest to use the new style comments that work for doxygen in both root 5 and 6. In that system, `//!` -> `//!<!` (and for streaming objects, `//` -> `///<`)

**Comment by raymondEhlers** [comment]
+test

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetCoreEmcal.h (hunk 1 above)
Comment (jaimenorman) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetCoreEmcal.h]: Thanks for the suggestion Raymond, will do. For the moment, can you approve as it is (assuming testing succeeds), and I will fix as suggested later?

Comment (jaimenorman) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetCoreEmcal.h]: Yes thanks for spotting! I also did the doxygen compatibility fixes where needed (now just on those which already have comments and transient members)

**Comment by raymondEhlers** [comment]
+1

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetCoreEmcal.h [hunk 2]
```diff
@@ -98,25 +99,31 @@ class AliAnalysisTaskJetCoreEmcal : public AliAnalysisTaskEmcalJet {
 	Float_t fJetEtaMax;
 	Float_t fJetHadronDeltaPhi;
 	TString fJetContName;
+	Bool_t fRunAnaAzimuthalCorrelation;
 	//
 	TRandom3 *fRandom;
```
Comment (raymondEhlers) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetCoreEmcal.h]: You should probably specify this and above variables as streaming, but this would be fine in another PR

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetCoreEmcal.h (hunk 1 above)
Comment (raymondEhlers) [reply PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetCoreEmcal.h]: Thanks!
